code/modules/admin/watchlist.dm

```diff
@@ -0,0 +1,118 @@
+client/proc/watchlist_add(target_ckey, browse = 0)
```

Comment:
All of these should be absolute pathed `/client/proc/watchlist_add(target_ckey, browse = 0)`